### ISSUE TITLE
More cores.

### ISSUE THREAD

**Issue by ahmeteminkocal** [meefik/linuxdeploy#564]
Im using it on my 8-core tablet. And its looks like just running on 4 of them. We need able to set how much cores can the machine use.

**Comment by ShapeShifter499** [comment]
This should be using the kernel of the device so unless the kernel you have is incorrectly compiled it should have access to all the cores. Perhaps whatever you are doing within the Chroot isn't requiring all too many cores?

**Comment by ahmeteminkocal** [comment]
@ShapeShifter499 yeah you were right. But I think it creates some problems as normally all the cores be online whether they are needed or not. Some programs don't recognize these cores.

**Comment by ShapeShifter499** [comment]
@ahmeteminkocal this isn't my app. I'm just another user who wanted to answer a question.

It's possible whatever programs your using could have issues running inside a chroot or possibility it is a bug (issue with the code). I'd try to submit a report or at least find a forum where someone might report back on what exactly is the issue.  

Also not every program magically uses all cores. The program usually needs to be created to use multiple cores when available.
